### ISSUE TITLE
Name formatting

### ISSUE THREAD

**Issue by jonthegeek** [kevin-m-kent/youtubeR#7]
(implement via api2r)

For readability, I like to standardize everything to `snake_case` in R packages. However, this API (and most APIs?) uses `camelCase`. The {snakecase} package handles conversion nicely, but we should implement something to check nested lists all the way down to make sure all names are in camelCase (unless perhaps there's some sort of flag saying this one is weird, if there's ever a weird case where we DON'T want this).

**Comment by kevin-m-kent** [comment]
hey - are you saying here that we should do camel case for all our functions, arguments for this package? And the conversion is just from the youtube API docs to our functions? Or that we should convert our functions to match?

**Comment by jonthegeek** [comment]
No, the opposite basically. We should stick with the more standard snake_case everywhere, but make sure things are back to their "native" format when the objects are sent back. There shouldn't be anything needed here yet, {nectar} will handle this.
